genai-perf/tests/test_exporters/test_console_exporter.py

```diff
@@ -124,10 +124,10 @@ def test_nonstreaming_llm_output(self, monkeypatch, capsys) -> None:
         )
         stats = Statistics(metrics=metrics)
 
-        config = ExporterConfig()
-        config.stats = stats.stats_dict
-        config.metrics = stats.metrics
-        config.args = args
+        assert isinstance(stats.metrics, Metrics)
```

Comment:
These asserts are necessary because the current architecture means that stats.metrics can be Metrics or TelemetryMetrics. It is beyond the scope of this PR to separate the two, but we do need asserts like this to check the correct type is used and to not result in type checking failures. It was previously opaque that we were using two types in the exporter config. We could always tackle it in a future cleanup effort if it causes issues, but making the typing transparent should help.